Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1636,7 +1631,21 @@ extension BrowserViewController: TabManagerDelegate {
             ReaderModeHandlers.readerModeCache = readerModeCache
 
             scrollController.browser = selected
+
+            // before we add a new webView, ensure that there aren't too many subviews already in the webViewContainer
+            if webViewContainer.subviews.count > 20,
+                let bottomWebView = ( webViewContainer.subviews.find {
```

Comment:
Does this actually return the bottom webView? I feel like this would return the first webView that find encounters [1]. What about:

```
let bottomWebView = webViewContainer.subviews.filter { $0.isKindOfClass(WKWebView) } .last
```

[1] https://github.com/mozilla/firefox-ios/blob/master/Utils/Extensions/ArrayExtensions.swift#L12